examples/smpi/replay_multiple_manual_deploy/replay_multiple_manual.cpp

```diff
@@ -0,0 +1,337 @@
+/* Copyright (c) 2009-2016. The SimGrid Team.
+ * All rights reserved.                                                     */
+
+/* This program is free software; you can redistribute it and/or modify it
+ * under the terms of the license (GNU LGPL) which comes with this package. */
+
+#include <algorithm>
+#include <fstream>
+#include <iostream>
+#include <regex>
+#include <stdexcept>
+#include <vector>
+
+#include <boost/algorithm/string.hpp>
+
+#include <simgrid/msg.h>
+#include <smpi/smpi.h>
+
+#include <libgen.h>
+
+using namespace std;
```

Comment:
nope, please don't.

Reply:
I'm just saying that we don't use "using namespace" constructs in SimGrid.

Reply:
Done in 397aaeeeb.